### PR TITLE
[Draft] Added L2 Normalize to frontend

### PR DESCRIPTION
**Pull request by AShedko**
Added L2Normalize to Frontend.
Tracker Issue: #1208

Signed-off-by: a.shedko sirius.frozen@gmail.com

### CODE REVIEW

File: compiler/luci/lang/include/luci/IR/Nodes/CircleL2Normalize.h [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+/**
+ * @brief L2NORMALIZE in Circle
+ */
+class CircleL2Normalize final : public FixedArityNode<1, CircleNodeImpl<CircleOpcode::L2NORMALIZE>>,
+                        public LuciNodeMixin<LuciNodeTrait::FusedActFunc>
+{
+public:
+  loco::Node *x(void) const { return at(0)->node(); }
+  void x(loco::Node *node) { at(0)->node(node); }
```
Comment (seanshpark) [inline compiler/luci/lang/include/luci/IR/Nodes/CircleL2Normalize.h:37]: From document, https://www.tensorflow.org/versions/r1.15/api_docs/python/tf/math/l2_normalize, how does `axis` and `epsilon` convert to TFlite?

Comment (AShedko) [reply compiler/luci/lang/include/luci/IR/Nodes/CircleL2Normalize.h:37]: No. Tflite version of this op only supports a fused activation.
Should we support axis and epsilon if backends do not support it?

**Comment by AShedko** [comment]
@nnfw-bot test format-checker

File: compiler/tflchef/core/src/Op/L2Normalize.h [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+private:
+  const tflchef::Operation *_operation;
+};
+
+struct L2NormalizeFactory final : public OpChefFactory
+{
+  std::unique_ptr<OpChef> create(const tflchef::Operation *operation) const override;
+};
+
+#endif // __OP_ABS_H__
```
Comment (d-krylov) [inline compiler/tflchef/core/src/Op/L2Normalize.h:49]: Fix comment, please.

File: compiler/luci/lang/include/luci/IR/Nodes/CircleL2Normalize.h (hunk 1 above)
Comment (seanshpark) [reply compiler/luci/lang/include/luci/IR/Nodes/CircleL2Normalize.h:37]: Thanks for the explanation and the graph!

Comment (seanshpark) [reply compiler/luci/lang/include/luci/IR/Nodes/CircleL2Normalize.h:37]: > Should we support axis and epsilon if backends and tflite do not support it?

Nope. We should go as-is with schema `L2NormOptions`

**Comment by seanshpark** [comment]
Looks good! Let's proceed with PRs;

**Comment by seanshpark** [comment]
@binarman , can we close this?

**Comment by binarman** [comment]
Everything is done, thank you!

**Pull request closed without merging**